### PR TITLE
OCM-6387 RHCS: Setup rosa-hcp-ad jobs

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/OCM-6387

depends on https://github.com/openshift/release/pull/49638

### CODE REVIEW

**Comment by radtriste** [comment]
@xueli181114 @yasun1 to you for approval

File: ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml [hunk 1]
```diff
@@ -240,6 +284,50 @@ tests:
...
+    env:
+      CASE_LABEL_FILTER: day1-negative&&!NonHCPCluster&&!Exclude
+      CHANNEL_GROUP: candidate
+      CLUSTER_PROFILE: rosa-hcp-ad
+      QE_USAGE: rosa-hcp-advanced-day1-negative-optional-presubmit
+      REGION: us-west-2
+      RHCS_ENV: staging
+      WAIT_OPERATORS: "false"
+    test:
+    - ref: rhcs-e2e-general-tests
```
Comment (yasun1) [inline ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml:330]: are there negative cases create some resources? If yes and the cases fail, when/where to release the resources?

Comment (radtriste) [reply ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml:330]: @yasun1 It should all be handled into the test case code directly.

Comment (radtriste) [reply ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml:330]: @yasun1 I created https://issues.redhat.com/browse/OCM-6838 to enable day1-negative. For now I am going to remove those jobs from here

**Comment by radtriste** [comment]
@yasun1 @xueli181114 Can you have a final look ?

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-critical-high-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-optional-presubmit

**Comment by radtriste** [comment]
looks like something changed since the last time I tested ... Gonna check this

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-critical-high-presubmit

**Comment by radtriste** [comment]
Fix in TF provider repository: https://github.com/terraform-redhat/terraform-provider-rhcs/pull/583

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-critical-high-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-critical-high-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-critical-high-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-critical-high-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-critical-high-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-critical-high-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-critical-high-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-critical-high-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-optional-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-optional-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-optional-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-critical-high-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-optional-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-critical-high-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-critical-high-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-advanced-optional-presubmit

**Pull request merged**